content/pt/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -0,0 +1,137 @@
+  de eleição de membro interno
+
+Como deve um Operador parecer em mais detalhe? Aqui está um exemplo em mais
+detalhe:
+
+1. Um recurso personalizado (*custom resource*) chamado SampleDB, que você pode
+  configurar para dentro do *cluster*.
+2. Um *Deployment* que garante que um *Pod* está a executar que contém a
+  parte controlador do operador.
+3. Uma imagem do *container* do código do operador.
+4. Código do controlador que consulta o plano de controle para descobrir quais
+  recursos *SampleDB* estão configurados.
+5. O núcleo do Operador é o código para informar ao servidor da API (*API server*) como fazer
+   a realidade coincidir com os recursos configurados.
+   * Se você adicionar um novo *SampleDB*, o operador configurará *PersistentVolumeClaims*
+     para fornecer armazenamento de base de dados durável, um *StatefulSet* para executar *SampleDB* e
+     um *Job* para lidar com a configuração inicial.
+   * Se você apagá-lo, o Operador tira um *snapshot* e então garante que
+     o *StatefulSet* e *Volumes* também são removidos.
+6. O operador também gere backups regulares da base de dados. Para cada recurso *SampleDB*,
+  o operador determina quando deve criar um *Pod* que possa se conectar
+   à base de dados e faça backups. Esses *Pods* dependeriam de um *ConfigMap*
+   e / ou um *Secret* que possui detalhes e credenciais de conexão com à base de dados.
+7. Como o Operador tem como objetivo fornecer automação robusta para o recurso
+   que gere, haveria código de suporte adicional. Para este exemplo,
+   O código verifica se a base de dados está a executar uma versão antiga e, se estiver,
+   cria objetos *Job* que o atualizam para si.
+
+## Instalar Operadores
+
+A forma mais comum de instalar um Operador é a de adicionar a
+definição personalizada de recurso (*Custom Resource Definition*) e
+o seu Controlador associado ao seu cluster.
+O Controlador vai normalmente executar fora do
+{{< glossary_tooltip text="plano de controle" term_id="control-plane" >}},
+como você faria com qualquer aplicação containerizada.
+Por exemplo, você pode executar o controlador no seu cluster como um *Deployment*.
+
+## Usando um Operador {#using-operators}
+
+Uma vez que você tenha um Operador instalado, usaria-o adicionando, modificando
+ou apagando a espécie de recurso que o Operador usa. Seguindo o exemplo acima,
+você configuraria um *Deployment* para o próprio Operador, e depois:
+
+```shell
+kubectl get SampleDB                   # encontra a base de dados configurada
+
+kubectl edit SampleDB/example-database # mudar manualmente algumas definições
+```
+
+&hellip;e é isso! O Operador vai tomar conta de aplicar
+as mudanças assim como manter o serviço existente em boa forma.
+
+## Escrevendo o seu prórpio Operador {#writing-operator}
+
+Se não existir no ecosistema um Operador que implementa
+o comportamento que pretende, pode codificar o seu próprio.
+[No que vem a seguir](#what-s-next) você vai encontrar
```

Comment:
This doesn't match the shortcode title. It should be:
```suggestion
[Qual é o próximo](#qual-é-o-próximo) você vai encontrar
```

(if you think “Qual é o próximo” is wrong, you can fix that  shortcode in a separate PR)